install.sh

```diff
@@ -0,0 +1,570 @@
+#! /bin/bash
+
+# USAGE:
+# export CONTAINER_TOOL=podman/docker
+# ./install.sh
+
+# Default container tool
+CONTAINER_TOOL=${CONTAINER_TOOL:-docker}
+
+# Check if an argument is provided
+if [[ "$#" -eq 1 ]]; then
+    if [[ "$1" == "docker" || "$1" == "podman" ]]; then
+        CONTAINER_TOOL="$1"
+    else
+        echo "Invalid argument: $1. Please use 'docker' or 'podman'."
+        exit 1
+    fi
+fi
+
+echo "===> Selected container tool: $CONTAINER_TOOL"
+
+
+# Extract the version number from pkg/config/version.go by getting everything
+# inside the quotes. Use -n to supress printing each line, and p to print the
+# modified line.
+SIGLENS_VERSION=`\
+    curl  --silent "https://api.github.com/repos/siglens/siglens/releases/latest" |
+    grep '"tag_name":' |
+    sed -E 's/.*"([^"]+)".*/\1/'`
+
+sudo_cmd=""
+
+# Text color
+RED_TEXT='\e[31m'
+GREEN_TEXT='\e[32m'
+RESET_COLOR='\e[0m'
+
+# Check sudo permissions
+if (( $EUID != 0 )); then
+    echo "===> Running installer with non-sudo permissions."
+    echo "     In case of any failure or prompt, run the script with sudo privileges."
+    echo ""
+else
+    sudo_cmd="sudo"
+fi
+
+os=""
+package_manager=""
+case "$(uname -sr)" in
+   Darwin*)
+     os="darwin"
+     package_manager="brew" ;;
+   Linux*amzn2*)
+     os="amazon linux"
+     package_manager="yum" ;;
+   Red\ Hat*|CentOS*|Fedora*)
+     os="linux"
+     package_manager="yum" ;;
+   Ubuntu*|Pop!_OS|Debian*|Linux*|Mint*)
```

Comment:
Should `Linux*|Mint*` be `Linux\ Mint*`, or should it be what you have? (I'm just looking at the diff with install_with_docker.sh)

Reply:
I changed this to how it is in the docker script. It is now changed to `Ubuntu*|Pop!_OS|Debian*|Linux*|Linux\ Mint*` But I do feel `Linux* | Linux\ Mint*` is redundant since `Linux*` covers the other Mint distributions also and we are assigning os as linux and package manager to `apt-get` 

So should I remove `Linux\ Mint*`?

Reply:
Oh I see, yeah as long as having `Linux*` does not include things we don't want included, you can just have `Linux*` and remove `Linux\ Mint*`

Reply:
@AndrewHess ignore the above comment.